### PR TITLE
Removes temporalAllowedPages from md.ts

### PR DESCRIPTION
**Pull request by corwintines**
## Description

- Removes the temporalAllowedPages list from md.ts

### CODE REVIEW

**Review by nhsz — APPROVED** [review]
LGTM!

File: src/lib/utils/md.ts [hunk 1]
```diff
@@ -279,19 +33,30 @@ const getPostSlugs = (dir: string, files: string[] = []) => {
...
+        if (process.env.ALLOWED_PAGES) {
+          const temporalAllowedPages = process.env.ALLOWED_PAGES.split(",")
+          // If it is a .md file (allowed content page), push the path to the files array
+          for (const page of temporalAllowedPages) {
+            const fullPagePath = join(CURRENT_CONTENT_DIR, page)
+
+            if (name.includes(fullPagePath)) {
+              files.push(
                 fullPagePath
                   .replace(CURRENT_CONTENT_DIR, "")
```
Comment (nhsz) [inline src/lib/utils/md.ts:45]: these lines are being repeated on the 2 cases, could be moved to another aux function inside `md.ts`
```ts
// ...
.replace(CURRENT_CONTENT_DIR, "")
.replace("/index.md", "")
// For replacing forward slashes generated in windows file paths
.replace(/\\/g, "/")
```

**Review by nhsz — CHANGES_REQUESTED** [review]
Great! leaving a small suggestion

**Comment by corwintines** [comment]
Some issues here with translations. Working to fix this. Ill convert to draft until this is ready.

**Comment by corwintines** [comment]
Will close this and reopen

**Pull request closed without merging**